tap_twitter_ads/schemas/cards.json

```diff
@@ -0,0 +1,37 @@
+{
+    "type": ["null", "object"],
+    "additionalProperties": false,
+    "properties": {
+      "name": {
+        "type": ["null", "string"]
+      },
+      "components": {
+        "type": ["null", "array"],
+        "items": {
+          "type": ["null", "object"],
+          "properties": {}
```

Comment:
Why did we keep empty properties here?

Reply:
There is no specific document available for the schema of the `cards` stream. In the SDK, they mentioned just the `components` field with the `list` datatype. That's why we kept empty properties here.